dev/.gitignore

```diff
@@ -1,5 +1,6 @@
 .data
 secrets.json
+db
```

Comment:
What is this for?

Reply:
to allow people to have a dev environment with a `.sqlite` db.

I've added a connection string to `secrets.json.example` with a connection string point to this directory.